modules/machineset-manually-scaling.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 // * machine_management/manually-scaling-machineset.adoc
+// * post_installation_configuration/cluster-tasks.adoc
```

Comment:
This is assuming that the cluster is an IPI install, UPI install does not have machinesets. Can we please explain this?